### DISCUSSION TITLE
个体资本论 - Miao Yu | 于淼

### DISCUSSION THREAD

**Discussion by giscus** [yufree/yufree.cn#436] [General]
# 个体资本论 - Miao Yu | 于淼

https://yufree.cn/cn/2022/05/30/personal-capitalist/

**Comment by KexingWestRoad** [comment]
很多人对很多事看不惯，本质上其实是对自己看不惯，自己的价值观与自己的主观能动性不匹配是很多个体内部矛盾的源泉。明明想做个早睡早起的人，睡前却打开手机说就刷一会；明明想考试得高分，但就是不愿意做练习题；明明想结交一个朋友，但就是发不出那个邀请链接……

一针见血，刀刀直击内心
在下佩服、佩服
看到你的博客，真的是找到了宝贝，在这么多纷杂的信息世界里，有这样一片信息宝藏之地，实属难得，读到你的一些见解，胜读10年书。
